### PR TITLE
Add functional tests for SmartOn pages

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds functional tests for SmartOn series pages

## Testing
Please make sure relevant tests pass in both Firefox and a non-Firefox browser (e.g. Chrome)

## Checklist
- [ ] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
@davehunt r?

File: tests/functional/smarton/test_all.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+from pages.smarton.base import SmartOnBasePage
+
+
+@pytest.mark.skip_if_firefox
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('slug', [
+    pytest.mark.smoke(('tracking')),
+    ('security'),
+    ('surveillance')])
```
Comment (davehunt) [inline tests/functional/smarton/test_all.py]: Not tested, but if you put these two tests in a class, you may be able to have the `parameterize` mark just on the class and avoid some duplication.

File: tests/pages/smarton/landing.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import BasePage
+
+
+class SmartOnLandingPage(BasePage):
```
Comment (davehunt) [inline tests/pages/smarton/landing.py]: Should this extend `SmartOnBasePage`?

**Comment by davehunt** [comment]
r+

**Pull request merged**